### PR TITLE
Add note about running multiple virt clusters

### PR DESCRIPTION
**Pull request by derekhiggins**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -226,6 +226,29 @@ This approach uses xinetd instead of iptables to allow IPv4 to IPv6 forwarding.
...
+## Hosting multiple dev-scripts on the same host
+
+dev-scripts has some support for running multiple instances on the
+same resources, when doing this CLUSTER\_NAME is used to namespace various
+resources on the virtual host. This support is not activly tested and
+has a few limitations but aims to allow you to run two separate clusters
+on the same host.
+
+To do this a the same user should be used to run dev-scripts for all
+environments but with a differant config file. In the config file at least
```
Comment (hardys) [inline README.md:238]: spelling s/differant/different/

Comment (derekhiggins) [reply README.md]: thanks, fixed

**Review by dhellmann — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**